features/setting_constraints/matching_arguments.feature

```diff
@@ -122,3 +122,28 @@ Feature: Matching arguments
       """
     When I run `rspec responding_differently_spec.rb`
     Then the examples should all pass
+
+  Scenario: Expecting a method invocatin with a block
```

Comment:
"invocation"

Reply:
Thanks, fixed in 9bc5f79928d236d91987cbeca2adc42a173d53ec